[ENH] Update info on assessment availability and following data model updates (#123)
neurobagel/documentation: docs/updating_dataset.md

_When in doubt, rerun both CLI commands._

### If only the subjects have changed
If subjects have been added to or removed from the dataset but the phenotypic TSV is otherwise unchanged (i.e., only new or removed rows, without changes to the available variables), you will need to:

- **Generate a new graph-ready data file** for the dataset by [re-running the CLI](cli.md) (`pheno` and `bids` steps) on your updated TSV and existing data dictionary

## Following a change in the _Neurobagel data model_

> As Neurobagel refines the way that specific subject properties are modeled as graph data, new tool releases will occasionally introduce breaking changes to the data model for subject-level information in a `.jsonld` graph data file.

Reviewer: ```suggestion
As Neurobagel continues developing the data model, new tool releases may introduce breaking changes to the data model for subject-level information in a `.jsonld` graph data file. This will be highlighted in the release notes.
```

> _If you have already created `.jsonld` files for a Neurobagel graph database_ but want to update your graph data to the latest Neurobagel data model following such a change, the easiest way would be to [rerun the CLI](cli.md) on the existing data dictionaries and phenotypic TSVs for the dataset(s) in the graph.

Reviewer: ```suggestion
_If you have already created `.jsonld` files for a Neurobagel graph database_ but want to update your graph data to the latest Neurobagel data model following such a change, you can easily do so by [rerunning the CLI](cli.md) on the existing data dictionaries and phenotypic TSVs for the dataset(s) in the graph.
```
